pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml

```diff
@@ -44,21 +44,21 @@ state-facts: true
 
   %div
     %h2.open-jobs
-      In 
```

Comment:
My editor linted this file when I changed it -- the page it generates looks fine without the trailing spaces:

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/25372625/235259765-38b3b740-23a0-44c6-b406-2281bd2840c2.png">